src/components/shared/epic.tsx

```diff
@@ -0,0 +1,124 @@
+// ----- Imports ----- //
+import { css, SerializedStyles } from '@emotion/core';
+import { basePx, headlineFont } from 'styles';
+import { from } from '@guardian/src-foundations/mq';
+import { palette, brandAlt } from '@guardian/src-foundations';
+import React, { useState, useEffect, useRef } from 'react';
+import { nativeClient } from 'native/nativeApi';
+import { SvgArrowRightStraight } from "@guardian/src-svgs"
+import { ThemeProvider } from 'emotion-theming'
+import { Button, buttonReaderRevenue } from '@guardian/src-button'
+
+
+// ----- Styles ----- //
+
+const EpicStyles = (): SerializedStyles => css`
+        width: calc(100% - ${basePx(4)});
+        margin: ${basePx(1)};
+
+        ${from.wide} {
+            margin: ${basePx(1, 0)};
+        }
+
+        clear: both;
+
+        border-top: 1px solid ${brandAlt[400]};
+        background: ${palette.neutral[97]};
+        padding: ${basePx(1)};
+        font-family: 'Guardian Text Egyptian Web';
+        clear: left;
+
+        h1:first-of-type {
+            ${headlineFont}
+        }
+
+        button {
+            margin: ${basePx(0, 1, 1, 0)};
+        }
+
+        .button-container {
+            margin-top: 3rem;
+        }
+
+        svg {
+            margin-left: 8px;
+            margin-top: -4px;
+            vertical-align: middle;
+        }
+
+        mark {
+            background: ${brandAlt[400]};
```

Comment:
```suggestion
            background: ${brandAltBackground.primary};
```
Derived from @SiAdcock's [suggestion](https://github.com/guardian/apps-rendering/pull/202#discussion_r382655221) on a recent PR of mine 🙂.